### ISSUE TITLE
Update OS Packages in distroless/python2.7-debian10:latest to mitigate CVEs

### ISSUE THREAD

**Issue by markdenihan** [GoogleContainerTools/distroless#453]
In distroless/pythong2.7:debian10:latest, python 2.7.16-2 suffers the following published CVE's;

https://security-tracker.debian.org/tracker/CVE-2019-9948
https://security-tracker.debian.org/tracker/CVE-2019-9740
https://security-tracker.debian.org/tracker/CVE-2018-20852
https://security-tracker.debian.org/tracker/CVE-2019-9947
https://security-tracker.debian.org/tracker/CVE-2019-16056
https://security-tracker.debian.org/tracker/CVE-2019-16935
https://security-tracker.debian.org/tracker/CVE-2019-10160

Seems the python2.7-debian10 is not picking up the fixes for CVE's along side the Debian Buster release cycle, as these are fixed in 2.7.16-2+deb10u1.

EDIT:
python2.7-debian10:latest also suffers CVE-2019-17594 and CVE-2019-17595 in libncursesw6 6.1+20181013-2+deb10u1. Both are fixed in in Debian Buster in libncursesw6  6.1+20181013-2+deb10u2

https://security-tracker.debian.org/tracker/CVE-2019-17594
https://security-tracker.debian.org/tracker/CVE-2019-17595

Testing with gcr.io/distroless/python2.7-debian10@sha256:81a088a8c82979e37e5ae8079c4d081c8fee0b9aeba5c0813150b557d4218929 (debug from 28th October 2019);
```bash
# python
Python 2.7.16 (default, Apr  6 2019, 01:42:57) 
[GCC 8.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import gzip
>>> import shutil
>>> with gzip.open('/usr/share/doc/python2.7-minimal/changelog.Debian.gz') as f_in:
...     with open('/usr/share/doc/python2.7-minimal/changelog.Debian', 'wb') as f_out:
...             shutil.copyfileobj(f_in, f_out)
... 
>>> 
# cd /usr/share/doc/python2.7-minimal
# ls
README.Debian        changelog.Debian     changelog.Debian.gz  copyright
# cat changelog.Debian
python2.7 (2.7.16-2) unstable; urgency=high

  [ Matthias Klose ]
  * CVE-2019-9636. Fix issue #36216: Add check for characters in netloc that
    normalize to separators. Closes: #924073.
  * CVE-2019-9948. Fix issue #35907: Stop urllib exposing the local_file schema
    (file://).

  [ Dimitri John Ledkov ]
  * Bump Build-Depedency and Dependency of libssl-dev and libss1.1 to
    1.1.1 or higher. As TLS1.3 constants leak into ssl module, thus one
    shouldn't mix and match python2.7 & libssl1.1. LP: #1808476

 -- Matthias Klose <doko@debian.org>  Sat, 06 Apr 2019 03:42:57 +0200
...
```
Testing with python installed on debian:buster;
```bash
# gunzip /usr/share/doc/python2.7-minimalchangelog.Debian.gz
# head -n 50 /usr/share/doc/python2.7-minimalchangelog.Debian
# head -n 50 changelog.Debian
python2.7 (2.7.16-2+deb10u1) buster; urgency=medium

  * CVE-2018-20852
  * CVE-2019-10160
  * CVE-2019-16056 (Closes: #940901)
  * CVE-2019-16935
  * CVE-2019-9740
  * CVE-2019-9947

 -- Moritz Mühlenhoff <jmm@debian.org>  Fri, 11 Oct 2019 00:02:15 +0200

python2.7 (2.7.16-2) unstable; urgency=high

  [ Matthias Klose ]
  * CVE-2019-9636. Fix issue #36216: Add check for characters in netloc that
    normalize to separators. Closes: #924073.
  * CVE-2019-9948. Fix issue #35907: Stop urllib exposing the local_file schema
    (file://).

  [ Dimitri John Ledkov ]
  * Bump Build-Depedency and Dependency of libssl-dev and libss1.1 to
    1.1.1 or higher. As TLS1.3 constants leak into ssl module, thus one
    shouldn't mix and match python2.7 & libssl1.1. LP: #1808476

 -- Matthias Klose <doko@debian.org>  Sat, 06 Apr 2019 03:42:57 +0200
```

**Comment by evanj** [comment]
I think this is probably because of #452 : Images haven't been published since October 28th. I made commit 4b46598d51017ea6bd37ad02235818703c95f3f2 which included version `2.7.16-2+deb10u1` when I built the images locally.

**Comment by chanseokoh** [comment]
If https://github.com/GoogleContainerTools/distroless/issues/453#issuecomment-571672129 is true, then I believe this issue is already resolved. Closing.
